### PR TITLE
adding a new contributor profile

### PR DESCRIPTION
**Pull request by heyjiawei**
Hello! Trying out pull requests!

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1630,11 +1635,6 @@ Noida
 - Bio: Programmer
 - Github: [Udit Mittal](https://github.com/udit-001)
 
-#### Name: [Abhishek Bhatt](https://github.com/ab-bh)
```
Comment (StevenCopeland) [inline CONTRIBUTORS.md]: Why are you deleting this name??????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????? Sorry. The problem is that 5000000000 million other people are doing the same thing as you and it gets kind of repetitive.

**Comment by heyjiawei** [comment]
Thats weird. The same name still gets removed even though I double check it to be in the file before I commit and pull request :\
And its that specific developer too

**Comment by StevenCopeland** [comment]
Fixed it.

**Comment by Roshanjossey** [comment]
Hi  @heyjiawei, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions

**Pull request merged**